### PR TITLE
Add the source of the documentation

### PR DESCRIPTION
**Pull request by mliszcz**
This PR adds source of the docs back to tango source tarball.

Per [propsal](https://github.com/tango-controls/TangoSourceDistribution/issues/40#issuecomment-503738615) in #40, it is not integrated with autotools.

### CODE REVIEW

File: distribution.properties [hunk 1]
```diff
@@ -34,3 +34,4 @@ docs-ver=9.2.5
 #utils
 tango_admin=Release_1.14
 #mtangorest.server=rc4-2.9
+tango-doc=master
```
Comment (bourtemb) [inline distribution.properties]: Wouldn't it make sense to create a branch 9.3.3 in tango-doc git repo and point to that branch here instead of master?

Comment (mliszcz) [reply distribution.properties]: Sure it will. But I'm not sure if tango-doc is ready for branchout (see: tango-controls/tango-doc#276).

**Comment by bourtemb** [comment]
Thank you very much for this PR.
prepare-9.3.3 branch has already been merged into master.
Please use the master branch as target branch for this PR or create a prepare-9.3.3-rc2 branch.
Thanks.

**Comment by mliszcz** [comment]
Just checked tarball size. It is now 310 MB gzipped. The increase comes from downloadable attachments (`:download:`), embedded in source and compiled documentation, e.g.:

```
source/tools-and-extensions/built-in/astor/paper.rst
6:You may be also interested in an :download:`ICALEPCS paper <astor.pdf>`
```

```
tango-cs@tangobox:~/build/tango-9.3.3$ du -h -t10M doc/src/
16M     doc/src/source/development
30M     doc/src/source/tools-and-extensions/archiving/mambo
13M     doc/src/source/tools-and-extensions/archiving/src-materials
53M     doc/src/source/tools-and-extensions/archiving
65M     doc/src/source/tools-and-extensions
37M     doc/src/source/tutorials-and-howtos/tutorials
37M     doc/src/source/tutorials-and-howtos
132M    doc/src/source
132M    doc/src/
tango-cs@tangobox:~/build/tango-9.3.3$ du -h -t10M doc/html/
12M     doc/html/_downloads/be963ee33f0af91b71cfe737f2af336b
13M     doc/html/_downloads/8156d2c5cc3d0ea2ee809a531377e7fe
51M     doc/html/_downloads
45M     doc/html/_images
102M    doc/html/
```

File: distribution.properties (hunk 1 above)
Comment (mliszcz) [reply distribution.properties]: I've changed this to 9.3.3 after branch was created in tango-doc.

**Review by bourtemb — APPROVED** [review]
Looks good to me! Thanks!

**Pull request merged**